tests/reftests/verbose-on.test

```diff
@@ -0,0 +1,312 @@
+-> installed qux.1
+-> installed baz.1
+-> installed bar.1
+-> installed foo.1
+Done.
+### opam reinstall qux --verbose-on foo | unordered | sed-cmd echo
+The following actions will be performed:
+=== recompile 4 packages
+  - recompile bar 1 [uses baz]
+  - recompile baz 1 [uses qux]
+  - recompile foo 1 [uses bar]
+  - recompile qux 1
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+-> removed   foo.1
+-> removed   bar.1
+-> removed   baz.1
+-> removed   qux.1
+-> installed qux.1
+-> installed baz.1
+-> installed bar.1
++ echo "foo" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/foo.1)
+- foo
+-> compiled  foo.1
+-> installed foo.1
+Done.
+### opam reinstall qux --verbose-on bar | unordered | sed-cmd echo
+The following actions will be performed:
+=== recompile 4 packages
+  - recompile bar 1 [uses baz]
+  - recompile baz 1 [uses qux]
+  - recompile foo 1 [uses bar]
+  - recompile qux 1
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+-> removed   foo.1
+-> removed   bar.1
+-> removed   baz.1
+-> removed   qux.1
+-> installed qux.1
+-> installed baz.1
++ echo "bar" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/bar.1)
+- bar
+-> compiled  bar.1
+-> installed bar.1
+-> installed foo.1
+Done.
+### opam reinstall qux --verbose-on bar,qux | unordered | sed-cmd echo
+The following actions will be performed:
+=== recompile 4 packages
+  - recompile bar 1 [uses baz]
+  - recompile baz 1 [uses qux]
+  - recompile foo 1 [uses bar]
+  - recompile qux 1
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
++ echo "qux" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/qux.1)
+- qux
+-> compiled  qux.1
+-> removed   foo.1
+-> removed   bar.1
+-> removed   baz.1
+-> removed   qux.1
+-> installed qux.1
+-> installed baz.1
++ echo "bar" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/bar.1)
+- bar
+-> compiled  bar.1
+-> installed bar.1
+-> installed foo.1
+Done.
+### opam reinstall qux --verbose-on foo,bar,baz,qux | unordered | sed-cmd echo
+The following actions will be performed:
+=== recompile 4 packages
+  - recompile bar 1 [uses baz]
+  - recompile baz 1 [uses qux]
+  - recompile foo 1 [uses bar]
+  - recompile qux 1
+
+<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
++ echo "qux" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/qux.1)
+- qux
+-> compiled  qux.1
+-> removed   foo.1
+-> removed   bar.1
+-> removed   baz.1
+-> removed   qux.1
+-> installed qux.1
++ echo "baz" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/baz.1)
+- baz
+-> compiled  baz.1
+-> installed baz.1
++ echo "bar" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/bar.1)
+- bar
+-> compiled  bar.1
+-> installed bar.1
++ echo "foo" (CWD=${BASEDIR}/OPAM/verbose-on/.opam-switch/build/foo.1)
+- foo
+-> compiled  foo.1
+-> installed foo.1
+Done.
+### : unknown package
+### opam reinstall qux --verbose-on unknown-pkg | unordered
```

Comment:
On unknown package, nothing is printed, should we?

Reply:
I think the current behaviour is fine